### PR TITLE
Update for Scalar DL/DB 2.0

### PR DESCRIPTION
**Pull request by scalarindetail**
- Rename according to renamed packages
- Revise the interface of Scalar DL Functions

### CODE REVIEW

File: emulator/README.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Parameters:
 Options:
   -h, --help        print the help and exit
     -fa, --function_argument=<functionArgument>
-        the argument passed to UDF
+        the argument passed to the function
```
Comment (feeblefakie) [inline emulator/README.md]:
```suggestion
        the argument passed to function
```

File: emulator/build.gradle [hunk 2]
```diff
@@ -15,8 +15,8 @@ dependencies {
     compile group: 'com.google.guava', name: 'guava', version: '27.0.1-jre'
     compile group: 'com.google.inject', name: 'guice', version: '4.2.0'
     compile group: 'org.glassfish', name: 'javax.json', version: '1.1.4'
-    compile group: 'com.scalar-labs', name: 'scalardb', version: '1.2.1'
-    compile group: 'com.scalar-labs', name: 'scalardl-client-sdk', version: '1.3.0'
+    compile group: 'com.scalar-labs', name: 'scalardb', version: '2.0.0'
+    compile group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.0.3'
```
Comment (feeblefakie) [inline emulator/build.gradle]:
```suggestion
    compile group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.0.4'
```
Sorry !

**Comment by scalarindetail** [comment]
Thank you !

**Comment by scalarindetail** [comment]
I tested the code after using scalardl-java-client-sdk 2.0.4 and the test passed.

**Pull request merged**